src/app/hooks/useScopeParam.ts

```diff
@@ -10,38 +10,50 @@ export const useNetworkParam = (): Network | undefined => {
   return network as Network | undefined
 }
 
+type ScopeInfo = SearchScope & {
+  valid: boolean
+}
+
 /**
  * Use this in situations where we might or might not have a scope
  */
-export const useScopeParam = (): SearchScope | undefined => {
+export const useScopeParam = (): ScopeInfo | undefined => {
   const { network, layer } = useParams()
+  const error = useRouteError()
 
   if (network === undefined && layer === undefined) return undefined
 
-  if (network === undefined || layer === undefined) {
-    throw new Error(
-      'You must either specify both network and layer or none of them. You can not have one but not the other.',
-    )
-  }
-
-  const scope: SearchScope = {
+  const scope: ScopeInfo = {
     network: network as Network,
     layer: layer as Layer,
+    valid: true,
   }
 
-  if (!scope || !RouteUtils.getEnabledNetworks().includes(scope.network))
-    throw new AppError(AppErrors.UnsupportedNetwork)
+  if (network === undefined || layer === undefined) {
+    scope.valid = false
+    if (!error)
+      throw new Error(
+        'You must either specify both network and layer or none of them. You can not have one but not the other.',
```

Comment:
Why are we encoding this here? Isn't it a consequence of routes, not scope param, so add a 404 route instead